comps/embeddings/tei/langchain/embedding_tei.py

```diff
@@ -57,28 +57,33 @@ async def embedding(
         logger.info(input)
     if isinstance(input, TextDoc):
```

Comment:
I think we should also make `TextDoc` accept `List[str]` input ? (need ask feng confirm)

Reply:
done